src/main/java/org/folio/event/dto/ItemFields.java

```diff
@@ -8,7 +8,8 @@
 public enum ItemFields {
 
   ID("id"),
-  HOLDINGS_RECORD_ID("holdingsRecordId");
+  HOLDINGS_RECORD_ID("holdingsRecordId"),
+  EFFECTIVE_LOCATION_ID("effectiveLocationId");
```

Comment:
Included to update POL `searchLocationIds`, should be an analogous to Holding `permanentLocationId`